### DISCUSSION TITLE
Change ToolbarItem TextColor based on Theme

### DISCUSSION THREAD

**Discussion by Orgbrat** [VladislavAntonyuk/MauiSamples#435] [Q&A] [answered]
@VladislavAntonyuk  I have been studying your MauiShellCustomization sample and I have Android changing color in the CustomShellToolbarAppearanceTracker/SetAppearance method with:
`		if (Settings.Theme == AppTheme.Light)
		{
			toolbar.OverflowIcon.SetTint(ContextCompat.GetColor(Microsoft.Maui.ApplicationModel.Platform.CurrentActivity, Resource.Color.m3_ref_palette_black));
		}
		else
		{
			toolbar.OverflowIcon.SetTint(ContextCompat.GetColor(Microsoft.Maui.ApplicationModel.Platform.CurrentActivity, Resource.Color.m3_ref_palette_white));
		}`

But I do not understand how to do the same in Windows, iOS and MacCatalyst. I need to change the TextColor based on Light or Dark Theme (black or white textcolor).  Can you help with how I might change the colors in the needed platforms.

Thanks for all you help in these samples as they have been a great help in many things.

**Comment by VladislavAntonyuk** [comment, marked as answer]
Your app should automatically change color on AppTheme. It is done by MAUI.
I don't have the exact code, but the idea is to find the element (one of the children of the toolbar) and set the color. On iOS it is a TextColor (UIColor), on Windows - Label Color
